internal/storage/object.go

```diff
@@ -31,4 +31,5 @@ type MinObject struct {
 	MetaGeneration int64
 	Updated        time.Time
 	Metadata       map[string]string
+	IsGzipEncoded  bool
```

Comment:
I first wondered if `IsGzipEncoded` should be a separate member or an item in `Metadata` dict. But as `Metadata` and `ContentEncoding` are separate in struct `gcs.Object`, it is clear that the same should be done here.